salt/monitoring/prometheus/prometheus.yml

```
@@ -14,24 +14,17 @@ rule_files:
     - /etc/prometheus/rules.yml
 
 scrape_configs:
-  - job_name: hanadb
+  # in a multi-cluster scenario, add another job name called
+  #  hacluster-europe-02
+  - job_name: hacluster-europe-01
```

perhaps we can rename this better but Imho is ok

---

I'd say to propose just a simple numbered scheme, people may not even have geographical regions.
```suggestion
  - job_name: cluster-01
```